org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.tests/xtend-gen/org/eclipse/xtext/example/domainmodel/tests/FormatterTest.java

```diff
@@ -82,71 +72,62 @@ public void example1() {
+    _builder_1.newLine();
+    _builder_1.append("\t\t");
+    _builder_1.append("\"xx\"");
+    _builder_1.newLine();
+    _builder_1.append("\t");
+    _builder_1.append("}");
+    _builder_1.newLine();
+    _builder_1.append("}");
+    _builder_1.newLine();
+    this.assertFormattedTo(_builder, _builder_1);
   }
 
   /**
    * This example shows how to test property-dependent formatting behavior.
    */
   @Test
   public void example3() {
-    final Procedure1<FormatterTestRequest> _function = (FormatterTestRequest it) -> {
-      final Procedure1<MapBasedPreferenceValues> _function_1 = (MapBasedPreferenceValues it_1) -> {
-        it_1.<String>put(FormatterPreferenceKeys.indentation, " ");
-      };
-      it.preferences(_function_1);
-      StringConcatenation _builder = new StringConcatenation();
-      _builder.append("entity Foo {");
-      _builder.newLine();
-      _builder.append(" ");
-      _builder.append("op foo():String {");
-      _builder.newLine();
-      _builder.append("  ");
-      _builder.append("\"xx\"");
-      _builder.newLine();
-      _builder.append(" ");
-      _builder.append("}");
-      _builder.newLine();
-      _builder.append("}");
-      _builder.newLine();
-      it.setExpectation(_builder);
-      StringConcatenation _builder_1 = new StringConcatenation();
-      _builder_1.append("entity Foo {");
-      _builder_1.newLine();
-      _builder_1.append("\t");
-      _builder_1.append("op foo():String {");
-      _builder_1.newLine();
-      _builder_1.append("\t\t");
-      _builder_1.append("\"xx\"");
-      _builder_1.newLine();
-      _builder_1.append("\t");
-      _builder_1.append("}");
-      _builder_1.newLine();
-      _builder_1.append("}");
-      _builder_1.newLine();
-      it.setToBeFormatted(_builder_1);
+    StringConcatenation _builder = new StringConcatenation();
+    _builder.append("entity Foo {");
+    _builder.newLine();
+    _builder.append("\t");
+    _builder.append("op foo():String {");
+    _builder.newLine();
+    _builder.append("\t\t");
+    _builder.append("\"xx\"");
+    _builder.newLine();
+    _builder.append("\t");
+    _builder.append("}");
+    _builder.newLine();
+    _builder.append("}");
+    _builder.newLine();
+    StringConcatenation _builder_1 = new StringConcatenation();
+    _builder_1.append("entity Foo {");
+    _builder_1.newLine();
+    _builder_1.append(" ");
+    _builder_1.append("op foo():String {");
+    _builder_1.newLine();
+    _builder_1.append("  ");
+    _builder_1.append("\"xx\"");
+    _builder_1.newLine();
+    _builder_1.append(" ");
+    _builder_1.append("}");
+    _builder_1.newLine();
+    _builder_1.append("}");
+    _builder_1.newLine();
+    final Procedure1<MapBasedPreferenceValues> _function = (MapBasedPreferenceValues it) -> {
+      it.<String>put(FormatterPreferenceKeys.indentation, " ");
     };
-    this._formatterTestHelper.assertFormatted(_function);
+    this.assertFormattedTo(_builder, _builder_1, _function);
```

Comment:
@miklossy are you sure you committed all the generated files correctly? In my workspace, after using the new master with this PR merged, I get that this file should be

```java
    final Consumer<MapBasedPreferenceValues> _function = (MapBasedPreferenceValues it) -> {
```

instead of the committed

```java
    final Procedure1<MapBasedPreferenceValues> _function = (MapBasedPreferenceValues it) -> {
```

Reply:
i see these changes too. can you open a fixing pr

Reply:
https://github.com/eclipse/xtext/pull/3003